app/src/main/java/com/example/sopt_assignment_dabin/ui/OnBoardingActivity.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.sopt_assignment_dabin.ui
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.ui.setupActionBarWithNavController
+import com.example.sopt_assignment_dabin.R
+import com.example.sopt_assignment_dabin.data.local.Room.RoomDAO
+import com.example.sopt_assignment_dabin.data.local.Room.RoomHelper
+import com.example.sopt_assignment_dabin.databinding.ActivityOnBoardingBinding
+import com.example.sopt_assignment_dabin.util.MyUtil.getHelper
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class OnBoardingActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityOnBoardingBinding
+    private lateinit var navController: NavController
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityOnBoardingBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        //앱을 처음 실행하는 경우가 아니면 온보딩 화면 안뜨게 함
+        CoroutineScope(Dispatchers.IO).launch {
+            try {
+                if (getHelper().get().onBording == false) {
+                    send()
+                }
+            } catch (e: Exception) {
+                e.toString()
+            }
+        }
```

Comment:
```suggestion
        lifecycleScope(Dispatchers.IO).launch {
            try {
                if (getHelper().get().onBording == false) {
                    send()
                }
            } catch (e: Exception) {
                e.toString()
            }
        }
```

일반 코루틴스코프를 사용하면 생명주기에 관계없이 cancel이 안되기에 생명주기에 맞춰서 cancel할 수 있는 lifecycleScope을 적절히 활용해서 코루틴을 사용하시면 더 좋을 것 같습니다.

Reply:
lifecycleScope를 사용하면 생명주기가 끝나면 코루틴 작업도 자동으로 취소되는군요..! 
몰랐던 사실인데 또 하나 배우네요 감사합니다!